source/contracts/reporting/ReportingWindow.sol

```diff
@@ -394,8 +400,6 @@ contract ReportingWindow is DelegationTarget, ITyped, Initializable, IReportingW
 
     function privateAddMarket(IMarket _market) private onlyInGoodTimes afterInitialized returns (bool) {
         require(!markets.contains(_market));
-        require(!firstReporterMarkets.contains(_market));
-        require(!lastReporterMarkets.contains(_market));
```

Comment:
Why were these removed?  They seem to be valid checks.

Reply:
I believe there should be no condition, barring potentially critical bugs in the system, where a market could be missing from the `markets` list but in one of those two.